### PR TITLE
Added speech_tools package

### PR DESCRIPTION
**Pull request by peci1**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by peci1** [comment]
@conda-forge/help-c-cpp, ready for review!

This is my first conda-forge PR, so please check it thoroughly :) 

The ultimate goal is to provide `festival` text-to-speech package (https://packages.ubuntu.com/bionic/festival), but I wanted to start slowly with one of its dependencies.

**Comment by peci1** [comment]
@conda-forge-admin, please ping team

File: recipes/libestools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - ncurses
+  run:
+    - _openmp_mutex
```
Comment (wolfv) [inline recipes/libestools/meta.yaml:31]: hmm, do you really need thsi dependency? I don't think you are using `openmp` (I believe thsi would be automatically added when using https://conda-forge.org/docs/maintainer/knowledge_base.html#message-passing-interface-mpi)

Comment (peci1) [reply recipes/libestools/meta.yaml:31]: There was a CI error without it...

File: recipes/libestools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/festvox/speech_tools/archive/63ff01938f81443f5a294bc5f9ea6ac6ab38f6b0.tar.gz
+  sha256: 8781ab15ce13fcddcb7df8fd23755a5cb6e4ab7d1a9bdbd741128b990b425496
+
+build:
+  skip: True  # [not linux]
```
Comment (wolfv) [inline recipes/libestools/meta.yaml:20]: I personally prefer lower case `true/false` but I don't think YAML cares :)

Comment (peci1) [reply recipes/libestools/meta.yaml:20]: I always understood YAML as kind of  Python-bound :)

File: recipes/libestools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "libestools" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/festvox/speech_tools/archive/63ff01938f81443f5a294bc5f9ea6ac6ab38f6b0.tar.gz
```
Comment (wolfv) [inline recipes/libestools/meta.yaml:16]: Do they have tags or publish release tar balls somewhere else?

Comment (wolfv) [reply recipes/libestools/meta.yaml:16]: Eg. I have found these links: https://www.cstr.ed.ac.uk/downloads/festival/2.4/

**Comment by wolfv** [comment]
looks good!

File: recipes/libestools/meta.yaml (hunk 3 above)
Comment (peci1) [reply recipes/libestools/meta.yaml:16]: Unfortunately, no. No releases/tags on github. Their homepage contains a download section, but that contains only release 2.4 from 2014...

File: recipes/libestools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (Tobias-Fischer) [inline recipes/libestools/meta.yaml:1]: Can you please remove these comments?

Comment (peci1) [reply recipes/libestools/meta.yaml:1]: Ah, right, I forgot...

File: recipes/libestools/meta.yaml (hunk 1 above)
Comment (wolfv) [reply recipes/libestools/meta.yaml:24]: I really really don't think this is necessary. I would love to see that CI error

Comment (peci1) [reply recipes/libestools/meta.yaml:24]: This was the output of my local test. I can try to remove this line in this PR, too. Just let me know based on this log:

```
   INFO: sysroot: '/home/conda/staged-recipes/build_artifacts/libestools_1674923795129/_build_env/x86_64-conda-linux-gnu/sysroot/' files: '['/home/conda/staged-recipes/build_artifacts/libestools_1674923795129/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/zone.tab', '/home/conda/staged-recipes/build_artifacts/libestools_1674923795129/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/tzdata.zi', '/home/conda/staged-recipes/build_artifacts/libestools_1674923795129/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/right/Zulu', '/home/conda/staged-recipes/build_artifacts/libestools_1674923795129/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/right/WET']'
   INFO (libestools,lib/libestools.so.2.5): lib/libestbase.so.2.5 found in this package
   INFO (libestools,lib/libestools.so.2.5): lib/libeststring.so.2.5 found in this package
   INFO (libestools,lib/libestools.so.2.5): Needed DSO lib/libtinfo.so.6 found in conda-forge::ncurses-6.3-h27087fc_1
   INFO (libestools,lib/libestools.so.2.5): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libestools.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
  ERROR (libestools,lib/libestools.so.2.5): Needed DSO lib/libgomp.so.1 found in ['_openmp_mutex']
  ERROR (libestools,lib/libestools.so.2.5): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (libestools,lib/libestools.so.2.5): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libestools.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestools.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so.2.5): lib/libeststring.so.2.5 found in this package
   INFO (libestools,lib/libestbase.so.2.5): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libestbase.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so.2.5): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libestbase.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestools.so.2.5.1): lib/libestbase.so.2.5 found in this package
   INFO (libestools,lib/libestools.so.2.5.1): lib/libeststring.so.2.5 found in this package
   INFO (libestools,lib/libestools.so.2.5.1): Needed DSO lib/libtinfo.so.6 found in conda-forge::ncurses-6.3-h27087fc_1
   INFO (libestools,lib/libestools.so.2.5.1): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libestools.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
  ERROR (libestools,lib/libestools.so.2.5.1): Needed DSO lib/libgomp.so.1 found in ['_openmp_mutex']
  ERROR (libestools,lib/libestools.so.2.5.1): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (libestools,lib/libestools.so.2.5.1): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libestools.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestools.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so.2.5.1): lib/libeststring.so.2.5 found in this package
   INFO (libestools,lib/libestbase.so.2.5.1): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libestbase.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so.2.5.1): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libestbase.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libeststring.so): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libeststring.so): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libeststring.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libeststring.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so): lib/libeststring.so.2.5 found in this package
   INFO (libestools,lib/libestbase.so): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libestbase.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libestbase.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libeststring.so.2.5): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libeststring.so.2.5): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libeststring.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libeststring.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libeststring.so.2.5.1): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libeststring.so.2.5.1): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libeststring.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libeststring.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestools.so): lib/libestbase.so.2.5 found in this package
   INFO (libestools,lib/libestools.so): lib/libeststring.so.2.5 found in this package
   INFO (libestools,lib/libestools.so): Needed DSO lib/libtinfo.so.6 found in conda-forge::ncurses-6.3-h27087fc_1
   INFO (libestools,lib/libestools.so): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libestools.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
  ERROR (libestools,lib/libestools.so): Needed DSO lib/libgomp.so.1 found in ['_openmp_mutex']
  ERROR (libestools,lib/libestools.so): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (libestools,lib/libestools.so): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libestools.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestools.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
Traceback (most recent call last):
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 220, in <module>
    build_all(os.path.join(root_dir, "recipes"), args.arch)
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 118, in build_all
    build_folders(recipes_dir, folders, arch, channel_urls)
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 176, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 180, in build
    return build_tree(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3097, in build_tree
    packages_from_this = build(metadata, stats,
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2369, in build
    newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 1658, in bundle_conda
    files = post_process_files(metadata, initial_files)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 1504, in post_process_files
    post_build(m, new_files, build_python=python)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1319, in post_build
    check_overlinking(m, files, host_prefix)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1225, in check_overlinking
    return check_overlinking_impl(m.get_value('package/name'),
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1204, in check_overlinking_impl
    raise OverLinkingError(overlinking_errors)
conda_build.exceptions.OverLinkingError: overlinking check failed 
["  ERROR (libestools,lib/libestools.so.2.5): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)", "  ERROR (libestools,lib/libestools.so.2.5.1): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)", "  ERROR (libestools,lib/libestools.so): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)"]
Traceback (most recent call last):
  File "build-locally.py", line 101, in <module>
    main()
  File "build-locally.py", line 95, in main
    run_docker_build(ns)
  File "build-locally.py", line 33, in run_docker_build
    subprocess.check_call([script])
  File "/usr/lib/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['.scripts/run_docker_build.sh']' returned non-zero exit status 1.
```

**Comment by Tobias-Fischer** [comment]
Can you please follow https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp and add `libgomp` explicitly in the build section to see whether that helps?

**Comment by peci1** [comment]
Done and it works, too.

**Comment by Tobias-Fischer** [comment]
Cool :)

**Pull request merged**